src/assets/scss/_desktop_navigation.scss

```diff
@@ -88,6 +88,14 @@ $promo-width: 200px;
       background: $colorGrayscale-pebble;
       display: block;
     }
+    > li:nth-of-type(5) {
```

Comment:
@j2trumpet @riceguitar Should this be a fixed number 5?  Can you do last in list?